app/scripts/services/validateRegistrant.js

```diff
@@ -210,7 +210,9 @@ angular
         );
       };
 
-      const blockInRegistrantType = (block, registrant) => {
+      this.shouldShowForRegistrantType = (block, registrant) => {
+        // We store the registrantTypes on the block that should NOT show the question.
```

Comment:
This reads clearer to me. I also did not know this until today. Very unintuitive!

```suggestion
        // block.registrantTypes contains the ids of registrants that should NOT show the question.
```